### ISSUE TITLE
What are the elements that make a successful stitching of a panoramic image possible?

### ISSUE THREAD

**Issue by jayepep** [lukasalexanderweber/stitching#67]
Very good work, thank you for your continued support.
I have a question
What are the elements that make a successful stitching of a panoramic image possible?
Imagine a scenario where there is a building in the middle and there are about 8 surveillance cameras or UAV(unmanned aerial vehicle) around it, each taking pictures at a certain angle (which will circle around the building), what are the factors that need to be taken into account if you eventually want to achieve a successful stitching?
For example, what height do I need to place these cameras? What is the minimum number of cameras needed and how many degrees each one covers to achieve a perfect stitching?

**Comment by lukasalexanderweber** [comment]
that question is sadly out of scope of my maintainance work. I would suggest you to experiment with dummy images and in parallel check the literature. The homography model within this package expects a rotating single camera, the affine model plane images like a scan

**Comment by jayepep** [comment]
okay, Thanks to your reply!
